### PR TITLE
Verbosive logging for commit status setter

### PR DESCRIPTION
**Pull request by lanwen**
This change is [Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/137)

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java [hunk 1]
```diff
@@ -128,6 +128,11 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
             String message = result.getMsg();
             GHCommitState state = result.getState();
 
+            listener.getLogger().printf(
+                    "[GitHub Commit Status Setter] Going to set status %s on repos %s (sha:%7.7s) with context:%s%n",
```
Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java]: normally prefix should be short and name plugin and then the same UI name i.e. `[GitHub] [getDescriptor().getDisplayName()]`

Comment (lanwen) [inline src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java]: Yep, but display name is too long for now

Comment (KostyaSha) [inline src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java]: Could be renamed to `Set GitHub commit status [universal]` btw and then rename log output without `going blabla` as it already what `Descriptor.getDisplayName()` claim.

Comment (lanwen) [inline src/main/java/org/jenkinsci/plugins/github/status/GitHubCommitStatusSetter.java]: You are right, will do

**Pull request merged**